Dockerfile

```diff
@@ -2,7 +2,8 @@ FROM alpine
 MAINTAINER Anil Madhavapeddy <anil@recoil.org>
 RUN apk update && apk add openssh socat
 RUN mkdir /root/.ssh && \
-    chmod 700 /root/.ssh
+    chmod 700 /root/.ssh && \
+    echo "root:$(dd if=/dev/urandom bs=3 count=16 | tr -d '\n')" | chpasswd
```

Comment:
I tried this a few times and am getting strange characters, including linefeeds, in the generated password.

I'd rather play it safe and use something [like this](https://unix.stackexchange.com/a/230676/293089):
```
</dev/urandom tr -dc 'A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_`{|}~' | head -c 16
```